docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: '3.9'
+
+services:
+  symfony:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
+    container_name: symfony
+    restart: always
+    ports:
+      - 8080:80
+    volumes:
+      - .:/var/www/html
+      - /var/www/html/vendor
+    depends_on:
+      lutedb:
+        condition: service_healthy
+    command: /bin/bash -c "/var/www/html/docker/app-init.sh && apachectl -D FOREGROUND"
+
+  lutedb:
+    image: mysql:8.0.31
+    container_name: lutedb
+    restart: always
+    healthcheck:
+      test: ["CMD", "mysql", "--version"]
+      interval: 3s
+      timeout: 2s
+      retries: 15
+      start_period: 15s
+    ports:
+      - 3306:3306
+    env_file:
+      - .env
+    environment:
+      MYSQL_ROOT_PASSWORD: ${DB_PASSWORD}
+      MYSQL_DATABASE: ${DB_DATABASE}
+      # MYSQL_USER: ${DB_USER}
```

Comment:
Perhaps set `DB_HOSTNAME=lutedb` here, I think that that would override the .env *and* .env.local files.